includes/geo_types.rs

```diff
@@ -1,22 +1,29 @@
+// Point x,y coords are mapped as:
+// x = latitude
+// y = longitude
 impl From<Point> for GeoPoint {
     fn from(field: Point) -> Self {
         Self {
-            x: field.x(),
-            y: field.y(),
+            longitude: field.x(),
+            latitude: field.y(),
         }
     }
 }
 
 impl From<GeoPoint> for Point {
     fn from(field: GeoPoint) -> Self {
-        Self::new(field.x, field.y)
+        Self::new(field.longitude, field.latitude)
     }
 }
+
+// Coord x,y are mapped as:
+// x = latitude
+// y = longitude
 impl From<GeoPoint> for Coord {
     fn from(field: GeoPoint) -> Self {
         Coord {
-            x: field.x,
-            y: field.y,
+            x: field.longitude,
```

Comment:
This doesn't appear correct according to the comments where Coord.x = latitude

Reply:
Ah, yes, I forgot to remove those comments after some reconsidering.